climpred/loadutils.py

```diff
@@ -105,48 +73,57 @@ def open_dataset(name, cache=True, cache_dir=_default_cache_dir,
     Returns:
         The desired xarray dataset.
     """
-    if name.endswith('.nc'):
-        name = name[:-3]
-    # use aliases
-    if name in file_alias_dict.keys():
-        name = file_alias_dict[name]
-    longdir = _os.path.expanduser(cache_dir)
-    fullname = name + '.nc'
-    localfile = _os.sep.join((longdir, fullname))
-    md5name = name + '.md5'
-    md5file = _os.sep.join((longdir, md5name))
-
-    if not _os.path.exists(localfile):
-        # This will always leave this directory on disk.
-        # May want to add an option to remove it.
-        if not _os.path.isdir(longdir):
-            _os.mkdir(longdir)
-
-        if extension is not None:
-            url = '/'.join((github_url, 'raw', branch, extension, fullname))
-            _urlretrieve(url, localfile)
-            url = '/'.join((github_url, 'raw', branch, extension, md5name))
-            _urlretrieve(url, md5file)
-        else:
-            url = '/'.join((github_url, 'raw', branch, fullname))
-            _urlretrieve(url, localfile)
-            url = '/'.join((github_url, 'raw', branch, md5name))
-            _urlretrieve(url, md5file)
-
-        localmd5 = _file_md5_checksum(localfile)
-        with open(md5file, 'r') as f:
-            remotemd5 = f.read()
-        if localmd5 != remotemd5:
+    def get_datasets():
+        """Prints out available datasets for the user to load if no args are
+           given."""
+        for key in file_descriptions.keys():
+            print(f"'{key}': {file_descriptions[key]}")
+
+    if name is None:
+        get_datasets()
```

Comment:
A trick to lessen indentation downstream is to `return` immediately in the function.

Reply:
Good point. The indentation felt weird.